elastic/kibana
# [Stack Monitoring] support entsearch package

### Summary

Update stack monitoring to read from data streams created by the [enterprisesearch package](https://github.com/elastic/integrations/pull/4926)

### Testing
- build [enterprisesearch package](https://github.com/elastic/integrations/pull/4926)
- start stack: `elastic-package stack up -v -d --version 8.7.0-SNAPSHOT`; make sure your image contains [this change](https://github.com/elastic/elastic-agent/pull/2121)
- start enterprisesearch service: `elastic-package service up -v`
- install elasticsearch integration with variables
  - `hosts: http://host.docker.internal:9201; username: elastic; password: changeme`
- install enterprisesearch integration with variables
  - `hosts: http://host.docker.internal:3002; username: elastic; password: changeme`
- connect [local kibana](https://github.com/elastic/kibana/blob/2d893bf40b47864058065f8fd2998d63c87f65f9/x-pack/plugins/monitoring/dev_docs/how_to/work_with_packages.md#connecting-a-local-kibana)
- navigate to Stack Monitoring on the local kibana, enterprise search section shows up with populated graphs
